buildout.cfg

```diff
@@ -1,33 +1,8 @@
 [buildout]
-extends =
-    https://raw.github.com/collective/buildout.plonetest/master/test-4.x.cfg
-    https://raw.github.com/collective/buildout.plonetest/master/qa.cfg
-
+extends = https://raw.githubusercontent.com/collective/buildout.plonetest/master/test-5.x.cfg
```

Comment:
Ah, I see: the 4.3 tests on Travis pass, but that is only because they actually use Plone 5...
The `bin/buildout annotate` line shows that Products.CMFPlone 5.0.3.1 is being used.